spec/test-component/component.json

```diff
@@ -1,31 +1,31 @@
 {
   "actions": {
-    "basicEmit": {
-      "main": "./lib/actions/basicEmit.js"
-    },
     "bigAction": {
-      "main": "./lib/actions/bigAction.js"
-    },
-    "manyEmits": {
-      "main": "./lib/actions/manyEmits.js"
-    },
-    "circularJson": {
-      "main": "./lib/actions/circularJson.js"
-    },
-    "bigIntJson": {
-      "main": "./lib/actions/bigIntJson.js"
-    },
-    "invalidEventEmit": {
-      "main": "./lib/actions/invalidEventEmit.js"
-    },
-    "thrownError": {
-      "main": "./lib/actions/thrownError.js"
-    },
-    "manyErrors": {
-      "main": "./lib/actions/manyErrors.js"
-    },
-    "returnData": {
-      "main": "./lib/actions/returnData.js"
```

Comment:
Readd these to component.json pls, they're my tests

Reply:
Yours are in `spec-integration`?

Reply:
oh I see I didn't notice the earlier move